exonum/src/storage/proof_list_index/tests.rs

```diff
@@ -827,47 +830,34 @@ mod root_hash_tests {
     }
 }
 
-mod list_hash_tests {
-    use hex::FromHex;
-
-    use crypto::Hash;
-    use storage::{hash::HashTag, Database, ListProof, MemoryDB, ProofListIndex};
-
-    #[test]
-    fn proof_of_absence_single() {
-        let db = MemoryDB::new();
-        let mut fork = db.fork();
-        let mut list = ProofListIndex::new("absence", &mut fork);
+#[test]
+fn proof_of_absence_single() {
+    let db = MemoryDB::new();
+    let mut fork = db.fork();
+    let mut list = ProofListIndex::new("absence", &mut fork);
 
-        list.push(vec![1]);
-        list.push(vec![2]);
-        list.push(vec![3]);
-        list.push(vec![4]);
-        list.push(vec![5]);
+    list.push(vec![1]);
+    list.push(vec![2]);
+    list.push(vec![3]);
+    list.push(vec![4]);
+    list.push(vec![5]);
 
-        let root_hash =
-            Hash::from_hex("5ba859b4d1799cb27ece9db8f7a76a50fc713a5d9d22f753eca42172996a88f9")
-                .unwrap();
+    let root_hash =
+        Hash::from_hex("5ba859b4d1799cb27ece9db8f7a76a50fc713a5d9d22f753eca42172996a88f9").unwrap();
 
-        let non_existed_index = 6u64;
-        let proof = list.get_proof(non_existed_index);
+    let non_existed_index = 6u64;
+    let proof = list.get_proof(non_existed_index);
 
-        match proof {
-            ListProof::Absent(index, actual_list_hash) => {
-                assert_eq!(index, non_existed_index);
+    match proof {
+        ListProof::Absent(index, actual_list_hash) => {
+            assert_eq!(index, non_existed_index);
 
-                let expected_hash = HashTag::list_hash(list.len(), root_hash);
-                assert_eq!(expected_hash, actual_list_hash);
-                assert!(index >= list.len());
-            }
-            _ => {
-                panic!("Unexpected proof {:?}", proof);
-            }
+            let expected_hash = HashTag::hash_list(list.len(), root_hash);
+            assert_eq!(expected_hash, actual_list_hash);
+            assert!(index >= list.len());
+        }
+        _ => {
+            panic!("Unexpected proof {:?}", proof);
         }
-    }
-
-    #[test]
-    fn proof_of_absence_range() {
-        //TBD
```

Comment:
Did we abadon the idea of writing tests for this?

Reply:
I plan to write it after fixing sandbox tests.